src/styles/blocks/contact.scss

```diff
@@ -0,0 +1,175 @@
+.contact-us {
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  gap: 40px;
+
+  &__title {
+    font-size: 48px;
+    line-height: 48px;
+    font-weight: 700;
+    letter-spacing: -1.5px;
+    margin: 0 auto 58px;
+    position: relative;
+    color: #000;
+
+    &:after {
+      content: "";
+      width: 24px;
+      height: 23px;
+      position: absolute;
+      top: 100%;
+      left: 97px;
+      background-image: url("../images/crown.svg");
+      background-size: contain;
+      background-repeat: no-repeat;
+    }
+  }
+
+  &__items {
+    display: flex;
+    justify-content: space-between;
+    flex-direction: row;
+
+    @media (max-width: 927px) {
+      flex-direction: column;
+    }
+
+    &-1 {
+      display: flex;
+      flex-direction: column;
+      margin-bottom: 24px;
+
+      &-phone {
+        display: flex;
+        flex-direction: column;
+        font-size: 16px;
+        line-height: 27px;
+        font-weight: 400;
+        color: #3e3e3e;
+        margin-bottom: 33px;
+
+        &-title {
+          font-size: 14px;
+          line-height: 27px;
+          font-weight: 400;
+          color: #9fadc2;
+        }
+      }
+
+      &-email {
+        display: flex;
+        flex-direction: column;
+        font-size: 16px;
+        line-height: 27px;
+        font-weight: 400;
+        color: #3e3e3e;
+
+        &-title {
+          font-size: 14px;
+          line-height: 27px;
+          font-weight: 400;
+          color: #9fadc2;
+        }
+      }
+    }
+
+    &-2 {
+      display: flex;
+      flex-direction: column;
+      margin-bottom: 24px;
+      font-size: 16px;
+      line-height: 27px;
+      font-weight: 400;
+      color: #3e3e3e;
+
+      &-title {
+        font-size: 14px;
+        line-height: 27px;
+        font-weight: 400;
+        color: #9fadc2;
+      }
+    }
+
+    &-3 {
```

Comment:
What should `2` or `3` mean? These names are not meaningful at all, Try to change it